### ISSUE TITLE
There is a bug for JavaBean to Map

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vadbch/orika#80]
```
What steps will reproduce the problem?
I define a class like this:
public class JsonObject extends HashMap<String,Object>{...}.
Orika can not map JavaBean to JsonObject.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?
orika1.4.0;win7 jdk7

```

Original issue reported on code.google.com by `linan...@126.com` on 31 Jan 2013 at 2:24

**Comment by GoogleCodeExporter** [comment]
Original comment by `elaat...@gmail.com` on 8 Mar 2013 at 9:06
- Changed state: **Accepted**
- Added labels: **R144**

**Comment by GoogleCodeExporter** [comment]
Original comment by `elaat...@gmail.com` on 5 Jun 2013 at 9:32
- Added labels: **Milestone-R144**
- Removed labels: **R144**

**Comment by GoogleCodeExporter** [comment]
```
A fix is applied in the current master; please checkout the most recent 
1.4.3-SNAPSHOT version.
```

Original comment by `matt.deb...@gmail.com` on 17 Jun 2013 at 6:02
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.4.3 release.
```

Original comment by `matt.deb...@gmail.com` on 7 Jul 2013 at 4:03
- Changed state: **Fixed**
